website/blog/2020-03-24-version-0.62.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Announcing React Native 0.62 with Flipper
+author: Rick Hanlon
+authorTitle: React Native Core at Facebook
+authorURL: https://twitter.com/rickhanlonii
+authorImageURL: https://avatars3.githubusercontent.com/u/2440089?s=460&v=4
+authorTwitter: rickhanlonii
+category: announcements
+---
+
+Today we’re releasing React Native version 0.62 which includes support for Flipper by default.
+
+This release comes in the midst of a global pandemic. We’re releasing this version today to respect the work of hundreds of contributors who made this release possible and to prevent the release from falling too far behind master. Please be mindful of the reduced capacity of contributors to help with issues and prepare to delay upgrading if necessary.
```

Comment:
```suggestion
This release comes in the midst of a global pandemic. We’re releasing this version regardless to respect the work of hundreds of contributors who made this release possible and to prevent the release from falling too far behind master. Please be mindful of the reduced capacity of contributors to help with issues and prepare to delay upgrading if necessary.
```

Making it a bit less ephemeral. Don’t feel strong about it, though.

Reply:
My goal here _is_ to be ephemeral - readers in the future after the crisis should understand that the reference point is the day of release. The issue with "regardless" or "nevertheless" is that they imply "in spite of" but we're saying here that the release _is_ being affected by the crisis.

Reply:
Gotcha, thanks 👌